boost.sh

```diff
@@ -201,6 +205,11 @@ parseArgs()
                 BUILD_LINUX=1
                 ;;
 
+            -linux-cxx-11-abi)
+                BUILD_LINUX=1
+                USE_CXX11_ABI=1
```

Comment:
The USE_CXX11_ABI flag, when enabled, means `DO NOT use CXX11_ABI` this is very misleading.